nobodywho/unity/src/Runtime/NobodyWhoChat.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace NobodyWho
+{
+    public class Chat : MonoBehaviour
+    {
+        public Model model;
+
+        ChatWrapper wrapper = ChatWrapper.New();
+
+        [TextArea(15, 20)]
+        public string systemPrompt = "";
+
+        [Header("Configuration")]
+        public string stopWords = "";
+        public uint contextLength = 4096;
+        public bool useGrammar = false;
+
+        [TextArea(15, 20)]
+        public string grammar;
+
+        [Header("Events")]
+        public UnityEvent<string> onToken = new UnityEvent<string>();
```

Comment:
It's called `response_updated` and `response_finished` in godot, but it's called `onToken` and `onComplete` in Unity?

Maybe we should rename things, so they're called the same thing in all of our plugins?
That might actually make the documentation be somewhat usable across engines.

If there's a good reason to change the name of the godot signals, I'm good with doing that. Otherwise, I think we should stick with the existing names, to avoid having to release a breaking change.